scopes/code/ui/code-compare/code-compare.tsx

```diff
@@ -36,17 +36,49 @@ export function CodeCompare({ fileIconSlot, className, CodeView = CodeCompareVie
 
   const { fileTree: baseFileTree = [], mainFile } = useCode(base?.model.id);
   const { fileTree: compareFileTree = [] } = useCode(compare?.model.id);
+  const fileCompareDataByName = componentCompareContext?.fileCompareDataByName;
 
-  const fileTree = uniq(baseFileTree.concat(compareFileTree));
+  const anyFileHasDiffStatus = useRef<boolean>(false);
```

Comment:
this is just to check if you went through the file tree sort?

Reply:
yes to know whether to default to index or to the first file with a diffStatus